modules/ding_app_variables/ding_app_variables.install

```diff
@@ -0,0 +1,13 @@
+<?php
+
+/**
+ * @file
+ * Install, update and uninstall functions for the Ding eKurser module.
+ */
+
+/**
+ * Removing unused 'ting_search_result_message_limit' variable.
+ */
+function ding_app_variables_update_7001() {
+  variable_delete('ting_search_result_message_limit');
+}
```

Comment:
Maybe this update hook should have been in ting_search as that was the module that had the admin form that used/set the variable.

Reply:
Good point - Fixed in 693052c